lib/query.js

```diff
@@ -44,6 +48,47 @@ export const isNumber = node => {
     }
 }
 
+export const isInteger = node => {
+    return isNumber(node) && Number.isInteger(Number(node.value))
+}
+
+export const isConstantFraction = node => isFraction(node) && node.args.every(arg => isNumber(arg))
+export const isIntegerFraction = node => isFraction(node) && node.args.every(arg => isInteger(arg))
```

Comment:
This can be simplified by doing `isFraction(node) && node.args.every(isInteger)`.  A similar change change be made to `isConstantFraction`.